app/cdap/components/ConnectionList/Components/ConnectionTabs/index.tsx

```diff
@@ -129,8 +130,7 @@ export default function ConnectionsTabs({
                 }
                 value={connectorType.name}
                 disableTouchRipple
-                key={`${connectorType.name}=${connectorTypeIndex}`}
-                id={connectorType.name}
+                key={`${connectorType.name}-${connectorTypeIndex}`}
```

Comment:
`key` should not include an index. Is `connectorType.name` not enough?

Reply:
As these are the file names, the file names can repeat in the list
in order to have a unique key, we had to use indices